examples/websocket/example.rst

```diff
@@ -99,7 +100,42 @@ You can start an interactive session with the socket as follows:
 
 Type ``Ctrl-c`` to exit the socket session.
 
-Step 4: Test proxying ``wss`` -> ``wss``
+Step 4: Test proxying ``ws`` -> ``ws`` on specific route
+********************************************************
+
+The proxy listening on port ``15000`` terminates the WebSocket connection without ``TLS`` on specific route ``/ws`` and then proxies
+to an upstream socket, also without ``TLS``.
+
+In order for Envoy to terminate the WebSocket connection, the
+:ref:`upgrade_configs <envoy_v3_api_msg_config.route.v3.RouteAction.UpgradeConfig>`
+in :ref:`RouteAction <envoy_v3_api_msg_config.route.v3.RouteAction>`
+must be set, as can be seen in the provided :download:`ws -> ws configuration <_include/websocket/envoy-ws-route.yaml>`:
+
+.. literalinclude:: _include/websocket/envoy-ws-route.yaml
+   :language: yaml
+   :lines: 1-35
+   :linenos:
+   :emphasize-lines: 24-25
```

Comment:
```suggestion
   :language: yaml
   :lines: 19-25
   :linenos:
   :emphasize-lines: 6-7 
   :caption: :download:`envoy-ws-route.yaml <_include/websocket/envoy-ws-route.yaml>`
   
```


rather show a short snippet with highlight and let the user download the full file more easily

Reply:
fixed